docs/reference/contributing/guidelines/workflow.md

```diff
@@ -1,45 +1,52 @@
+Responsibilities:
 
-#### Reporting and fixing bugs
+1. Check for CLA compliance.
+1. Ensure the relevant stakeholders review pull requests.
+1. Guide contributors both technically and procedurally.
+1. Run pull requests through the CI systems.
+1. Put label version.
+1. Merge pull requests into the requested branches.
+1. Make periodic patch and feature releases.
 
-Before submitting a bug report or a bug fix, please [discuss it on the forums](https://os.mbed.com/forum/bugs-suggestions/) to avoid duplication of work, as we or others might be working on it already.
+The current maintainers are:
 
-##### Bug reports (issues) on GitHub
+* [Anna Bridge](https://os.mbed.com/users/AnnaBridge).
+* [Martin Kojtal](https://os.mbed.com/users/Kojto).
+* [Jimmy Brisson](https://os.mbed.com/users/theotherjimmy).
+* [Shrikant Tudavekar](https://os.mbed.com/users/shrikant1213).
+* [Sam Grove](https://os.mbed.com/users/sam_grove).
+* [Cruz Monrreal](https://os.mbed.com/users/MrCruz).
 
-All Mbed OS is on GitHub; please use GitHub's [issues mechanism](https://guides.github.com/features/issues/) to open a bug report directly against the relevant GitHub repository.
+#### Contributions
 
-##### Bug fixes
+Before contributing an enhancement (new feature, new port and so on), please [discuss it on the forums](https://os.mbed.com/forum/bugs-suggestions/) to avoid duplication of work, as we or others might be working on a related feature.
 
-Please refer to the [code contributions chapter](/docs/v5.7/reference/guidelines.html#style).
+We can only accept contributions through GitHub if you create a pull request from forked versions of our repositories. This allows us to review the contributions in an easy-to-use and reliable way, under public scrutiny.
 
-A member of the Mbed team must verify bug fixes before we pull the fixes into the main branch. You must therefore use GitHub to fork the repo and then submit a pull request with your changes.
+Please create separate pull requests for each concern; each pull request needs a clear unity of purpose. In particular, separate code formatting and style changes from functional changes. This makes each pull request’s true contribution clearer, so review is quicker and easier.
 
-The last line in your commit message description should say “Fixes #deadbeef”, where “deadbeef” is the issue number in GitHub. This allows GitHub to automatically close the issue when the commit is merged into the default branch.
+##### Reporting bugs
 
-#### Further reading
+Please submit all Mbed OS bugs [on the forums](https://os.mbed.com/forum/bugs-suggestions/).
 
-Please see the [code contributions chapter](/docs/v5.7/reference/guidelines.html#style) for the guidelines to GitHub pull requests and the coding style guide.
+The bug report should be reproducible (fails for others) and specific (where and how it fails). We will close insufficient bug reports.
```

Comment:
>The bug report should be reproducible (fails for others)

If it's an intermittent issue it may not be readily reproducible. 

Should we put more details what we expect to be in a report? Like versions, steps, example, expected and actual outcomes,...

Reply:
There should be enough information captured so that anyone trying to reproduce the bug can replicate the environment and code versions used

Reply:
> Should we put more details what we expect to be in a report? Like versions, steps, example, expected and actual outcomes,...

 Having `how to report a bug` would be nice to have, currently there is something in github template issue (not sufficient though). I'll talk to the team who will be handling issues